crates/compiler/can/src/expr.rs

```diff
@@ -715,7 +726,6 @@ pub fn canonicalize_expr<'a>(
 
             let output = Output {
                 references,
-                tail_call: None,
```

Comment:
These were removed during my attempts to fix tail calls, but didn't need to be. However, their default value is `None` anyway, so I kept them out.